lib/model.js

```diff
@@ -176,62 +182,99 @@ Model.new = function(name, schema, options, thinky) {
   return model
 }
 
-Model.prototype.isReady = function() {
-  return this._getModel()._tableReady;
-}
+/**
+ * Create the model's table.
+ * @return {Promise=} Returns a promise which will resolve when the table is ready.
+ */
+Model.prototype.tableReady = function() {
+  var self = this;
+  var model = this._getModel();
+  if (!this._initModel) return Promise.resolve();
+  if (this._tableReadyPromise) return this._tableReadyPromise;
+
+  // Create the table, or push the table name in the queue.
+  var r = model._thinky.r;
+  this._tableReadyPromise = model._thinky.dbReady()
+  .then(function() {
+    return r.tableList().contains(model._name).do(function(result) {
```

Comment:
See my [other comment](https://github.com/neumino/thinky/pull/278/files#r33728944). Same thing, I think we should just assume the table needs to be created and catch the error if it already exists.